main.go

```diff
@@ -452,7 +453,14 @@ func main() {
 		die("unable to read template from %v: %v", opts.TemplateFile, err)
 	}
 
-	templ, err := template.New("").Funcs(helperFuncs).Parse(string(buf))
+	FuncMap := sprig.GenericFuncMap()
```

Comment:
`FuncMap` should be `funcMap`, [unless your intention is to make it public](https://golang.org/ref/spec#Exported_identifiers)

Reply:
Ah, yes. of course